moped-database/migrations/default/1748534889272_create_notes_view/up.sql

```diff
@@ -0,0 +1,631 @@
+-- Update moped_note_types to have source column
+ALTER TABLE moped_note_types
+ADD COLUMN source text NOT NULL DEFAULT 'moped';
+
+-- Insert eCapris status note type that will be used in notes view
+INSERT INTO moped_note_types (name, slug, source)
+VALUES
+('eCapris Status Update', 'ecapris_status_update', 'ecapris');
+
+COMMENT ON COLUMN moped_note_types.source IS 'Source of the note type, e.g., Moped or eCapris applications';
+
+-- Create the combined_project_notes view for the project notes tab and status update in summary view
+CREATE OR REPLACE VIEW combined_project_notes AS
+SELECT
+    ('M' || moped_proj_notes.project_note_id) AS id,
+    moped_proj_notes.project_note_id AS original_id,
+    moped_proj_notes.project_note,
+    moped_proj_notes.created_at,
+    moped_proj_notes.created_by_user_id,
+    moped_proj_notes.project_id,
+    (moped_users.first_name || ' ' || moped_users.last_name) AS author,
+    moped_note_types.name AS note_type_name,
+    moped_note_types.slug AS note_type_slug,
+    moped_phases.phase_name AS phase_name,
+    moped_phases.phase_key AS phase_key,
+    moped_proj_notes.is_deleted,
+    moped_proj_notes.phase_id,
+    TRUE AS is_editable,
+    NULL AS ecapris_subproject_id
+FROM
+    moped_proj_notes
+LEFT JOIN moped_users ON moped_proj_notes.created_by_user_id = moped_users.user_id
+LEFT JOIN moped_note_types ON moped_proj_notes.project_note_type = moped_note_types.id
+LEFT JOIN moped_phases ON moped_proj_notes.phase_id = moped_phases.phase_id
+WHERE moped_proj_notes.is_deleted = FALSE
+UNION ALL
+SELECT
+    ('E' || ecapris_subproject_statuses.id) AS id,
+    ecapris_subproject_statuses.id AS original_id,
+    ecapris_subproject_statuses.sub_project_status_desc AS project_note,
+    ecapris_subproject_statuses.review_timestamp AS created_at,
+    ecapris_subproject_statuses.created_by_user_id,
+    NULL AS project_id,
+    COALESCE(
+        (moped_users.first_name || ' ' || moped_users.last_name),
+        CASE
+            WHEN ecapris_subproject_statuses.reviewed_by_name LIKE '%,%' 
+                THEN TRIM(SPLIT_PART(ecapris_subproject_statuses.reviewed_by_name, ',', 2)) || ' ' || TRIM(SPLIT_PART(ecapris_subproject_statuses.reviewed_by_name, ',', 1))
+            ELSE LOWER(ecapris_subproject_statuses.reviewed_by_email)
+        END
+    ) AS author,
```

Comment:
I was originally going to use `reviewed_by_email` as a fallback to eCapris statuses that didn't match an existing Moped user, but this column is not always populated in the eCapris database. 

I could not find any statuses without `reviewed_by_name` or statuses where the format was not `Last, First` but I added a check just in case we see something different in the future.